README.md

```diff
@@ -185,6 +185,38 @@ This tool checks for updates automatically, however this is cached for some time
 1. Check for an uncached update: `so self:update-check --force`.
 1. Update `so` to the latest version with: `so self:update`.
 
+### Add additional Top Level Domains (TLDs)
+
+By default, the SquareOne Docker DNS configuration maps all domain names that end with the TLD `.tribe` to itself. You can add additional TLDs as necessary for your projects.
+
+1. Create the file `~/.config/squareone/globaldocker-compose.override.yml`.
```

Comment:
```suggestion
1. Create the file `~/.config/squareone/global/docker-compose.override.yml`.
```

Also, the command `so config:compose-copy` does make a copy of the main global `docker-compose.yml` and saves it to the path above. But then they would need to clear out everything and then add your changes, so your call there what's easier to follow.

Reply:
I went back & forth on that. Since it's just a few specific entries that need to be modified, it was easier to start fresh than remove everything. On the flip side, using the compose-copy command does make sure that whatever values exist in the main compose will be accurate in the override. I really don't know which is easier. /shrug

Reply:
I have no preference either way :) Let's go with what you have now, just the path with my suggestion above needs to be correct.

Reply:
Sound good. Fixed & pushed the path typo. :)